install/conf/ipa.conf

```diff
@@ -20,6 +20,11 @@ DirectoryIndex index.html
 # requests, ticket #2767. This should easily support a 64KiB PAC.
 LimitRequestFieldSize 100000
 
+# Increase connection keep alive time. Default value is 5 seconds, which is too
+# short for interactive ipa commands. 30 seconds is a good compromise.
+KeepAlive On
+KeepAliveTimeout 30
```

Comment:
Your assumption is incorrect. For ```ipa``` client, the connection is automatically closed by ```ipaclient.api.Backend.rpcclient.disconnect()``` or when the client process exits. The 30 seconds keepalive timeout optimizes both ```ipa``` and browser sessions. For browser session I would even increase the keepalive timeout for 60 seconds in order to reduce the load on the server.

Modern webservers like Apache use high performance socket handling features of modern operating systems. ```epoll``` can easily handle thousands of connections. Repeating TCP handshake and TLS handshake over and over again cost magnitudes more performance than watching a bunch of additional sockets for I/O. A longer keepalive period won't be a problem until we have to handle thousands of client connections simultaneously.